### PR TITLE
sys-apps/dbus-broker and dependencies

### PR DESCRIPTION
**Pull request by floppym**
cc @gentoo/systemd

https://lists.freedesktop.org/archives/systemd-devel/2017-August/039448.html

### CODE REVIEW

**Comment by floppym** [comment]
cc @gentoo/freedesktop

File: dev-libs/c-dvar/c-dvar-1.ebuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+HOMEPAGE="https://github.com/c-util/c-dvar"
+SRC_URI="https://github.com/c-util/c-dvar/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+src_install() {
+	meson_src_install
+	doheader src/c-dvar-type.h
```
Comment (zx2c4) [inline dev-libs/c-dvar/c-dvar-1.ebuild]: Has the need for this file being installed been reported upstream?

File: profiles/features/selinux/package.mask [hunk 2]
```diff
@@ -34,3 +34,4 @@ net-misc/netctl
 sys-apps/gentoo-systemd-integration
 sys-apps/systemd-readahead
 sys-process/systemd-cron
+sys-apps/dbus-broker
```
Comment (zx2c4) [inline profiles/features/selinux/package.mask]: dbus-broker has selinux support:
https://news.ycombinator.com/item?id=15081662

File: dev-libs/c-dvar/c-dvar-1.ebuild (hunk 1 above)
Comment (floppym) [reply dev-libs/c-dvar/c-dvar-1.ebuild]: Yes, and it has been fixed there.

https://github.com/c-util/c-dvar/commit/e272c9681c453d565e8b626a5b0b2edc71e6d640

File: profiles/features/selinux/package.mask (hunk 2 above)
Comment (floppym) [reply profiles/features/selinux/package.mask]: Unfortunately, our selinux profiles mask systemd, so this package must also be masked.

File: sys-apps/dbus-broker/dbus-broker-3.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+IUSE="selinux test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/c-dvar-1
+	>=dev-libs/c-rbtree-2
+	>=dev-libs/expat-2.2
+	>=dev-libs/glib-2.50:2
+	>=sys-apps/systemd-230
+	selinux? ( sys-libs/libselinux )
```
Comment (floppym) [inline sys-apps/dbus-broker/dbus-broker-3.ebuild]: The meson build flag for this is being added upstream.

https://github.com/bus1/dbus-broker/pull/31

**Comment by floppym** [comment]
I think I will wait for v4 on this.

**Pull request closed without merging**